### ISSUE TITLE
Update Pipfile, Pipfile.lock

### ISSUE THREAD

**Issue by keikoro** [Zulko/moviepy#1350]
We [originally introduced these](https://github.com/Zulko/moviepy/pull/931#issuecomment-477056074) to allow for an alternative `pipenv`-based workflow, but we haven't kept them updated.

The following needs doing:

- [ ] find out how `pipenv` + necessary files were originally (meant to be) used with the project
- [ ] update `Pipfile`, `Pipfile.lock`
- [ ] document usage / update existing documentation (if available)

The last point might need checking back with maintainers if the workflow hasn't been documented yet, so we can decide together where best to put the info.

Tagging this as [good-first-issue](https://github.com/Zulko/moviepy/labels/good%20first%20issue) with the caveat that it would be very helpful if you knew your way around `pipenv`.

**Comment by keikoro** [comment]
Note that #1349 needs to be merged before it makes sense to work on this.

**Comment by tburrows13** [comment]
And now #1351 is relevant as well.

**Comment by tburrows13** [comment]
Unneeded as of #1416. If someone thinks that we should add them back, feel free to discuss here or in a new issue.
